types/src/v0/mod.rs

```diff
@@ -135,25 +137,39 @@ impl NodeType for SeqTypes {
     type BuilderSignatureKey = FeeAccount;
     type AuctionResult = SolverAuctionResults;
 }
+#[derive(Clone, Default, Debug, Copy)]
+pub struct SequencerVersions<Base: StaticVersionType, Upgrade: StaticVersionType> {
+    _pd: PhantomData<(Base, Upgrade)>,
+}
+
+impl<Base: StaticVersionType, Upgrade: StaticVersionType> SequencerVersions<Base, Upgrade> {
+    pub fn new() -> Self {
+        Self {
+            _pd: Default::default(),
+        }
+    }
+}
 
-#[derive(Debug, Copy, Clone)]
-pub struct SequencerVersions;
+pub type BaseV01UpgradeV02 = SequencerVersions<StaticVersion<0, 1>, StaticVersion<0, 2>>;
```

Comment:
Since we are defining aliases for these, it might be a little easier to keep in sync if we use the alias everywhere.

Reply:
this is used for test network mostly

Reply:
I think I should rename it to MockSequencerVersions